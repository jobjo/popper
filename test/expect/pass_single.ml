open Popper

let () = Clean.run @@ test @@ fun () -> pass
