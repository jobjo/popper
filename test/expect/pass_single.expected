
1/1 tests passed in XXs:

  âœ“  Anonymous  Passed

Test Successful: 1/1 tests passed in XXs.

